cartaya/profile-programers/resources/css/style.css

```diff
@@ -0,0 +1,117 @@
+@font-face {
+    font-family: 'Lato-Light';
+    src: url('./../font/Lato-Light.eot?#iefix') format('embedded-opentype'),
+         url('./../font/Lato-Light.woff') format('woff'),
+         url('./../font/Lato-Light.ttf') format('truetype'),
+         url('./../font/Lato-Light.svg#FontName') format('svg');
+}
+
+body {
+    align-items: center;
+    background-color: rgb(231, 234, 238);
+    display: flex;
+    height: 100vh;
+    justify-content: center;
+    margin: 0;
+}
+
+.lato-font {
```

Comment:
In this class, you only need use the basic styles of font. It should be maintainable easy.